notebooks/sedml.ipynb

```diff
@@ -13,7 +13,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This notebook creates a simple model in [SED-ML version 1 level 4](https://sed-ml.org/specifications.html)."
+    "## 1) Including libraries and helper functions"
```

Comment:
This new header at line 16 looks correct, but there is still the old, now duplicated header at line 32?

Reply:
Thanks - missed I'd duplicated it